test/alternator/test_streams.py

```diff
@@ -678,7 +678,6 @@ def do_updates(table, p, c):
 # Test OLD_IMAGE using UpdateItem. Verify that the OLD_IMAGE indeed includes,
 # as needed, the entire old item and not just the modified columns.
 # Currently fails in Alternator because the item's key is missing in OldImage (#6935).
```

Comment:
Please replace this line ("currently fails ...") with a statement that this reproduces issue #6935.

Reply:
fixed